src/utils.ts

```diff
@@ -0,0 +1,35 @@
+export type Donation = {
+  amount: number;
+  currencyCode: string;
+  receiptUrl: string;
+};
+
+export async function getDonations(): Promise<Donation[]> {
+  try {
+    const response = await fetch(
+      'https://donations-api.justgiving.com/v1/donations',
+      {
+        headers: {
+          'X-Secret-Key': 'eromynAterceSoStoN',
+        },
+      },
+    );
+
+    return response.json();
+  } catch (err) {
+    return [];
+  }
+}
+
+export function track(event: string) {
+  (window as any).track(event);
```

Comment:
Will error if `window.track` is not defined... should be
```suggestion
  (window as any)?.track(event);
```